### PR TITLE
Add current state of the document

### PR DESCRIPTION
**Pull request by albalitz**
I wrote this a while ago as a secondary project while doing other stuff, so this may contain typos and other errors... Plus this may not be entirely up to date.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,107 @@
...
+#### uBlock origin
+Firefox: https://addons.mozilla.org/en-US/firefox/addon/ublock-origin/  
+Chrome/Chromium based: https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm
+
+Source code availabe on GitHub: https://github.com/gorhill/uBlock
+
+
+### worth reading
+- [Why I left Facebook](https://www.blakewatson.com/journal/why-i-left-facebook/)  
+  A blog post showing you things you should think about when you use Facebook.
```
Comment (michel-zimmer) [inline README.md]: `[...] Facebook and other companies and products owned by Facebook.`

Comment (albalitz) [reply README.md]: Yes, that's probably better.

File: README.md [hunk 2]
```diff
@@ -1 +1,107 @@
 # tinfoil
```
Comment (michel-zimmer) [inline README.md]: Depending on whom we are addressing we should make this more readable - and the majority prefers upper case headlines etc.

Comment (michel-zimmer) [reply README.md]: (Not the repo itself, but inside the text)

File: README.md (hunk 1 above)
Comment (albalitz) [reply README.md]: @michel-zimmer done.

File: README.md (hunk 2 above)
Comment (albalitz) [reply README.md]: @michel-zimmer done.

File: README.md (hunk 1 above)
Comment (michel-zimmer) [reply README.md]: Wait, that sounds wrong now. `[...] Facebook and other Products from Facebook Inc. and Facebook Inc. owned companies.` This now also includes a distinction between the platform and the company.

Comment (michel-zimmer) [reply README.md]: Maybe include `[...] other products (Instagram, WhatsApp, ...)`?

Comment (albalitz) [reply README.md]: You're right. Changed it.
I also changed
> when you use

to
> when using

**Pull request merged**